<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeManage</class>
 <widget class="QWidget" name="TimeManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
   </property>
   <property name="text">
    <string>영업시간관리</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>120</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>110</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>메뉴 관리</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_order">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>주문확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>110</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>영업시간관리</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">ui-&gt;tableWidget-&gt;setStyleSheet(
    &quot;QTableWidget {&quot;
    &quot;   border: 1px solid #ccc;&quot;
    &quot;   border-radius: 15px;&quot;
    &quot;   background-color: white;&quot;
    &quot;}&quot;
    &quot;QHeaderView::section {&quot;
    &quot;   background-color: #f0f0f0;&quot;
    &quot;   border: none;&quot;
    &quot;}&quot;
);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>480</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
   </property>
   <property name="text">
    <string>설정하기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
